### PR TITLE
add readOnly flag

### PR DESCRIPTION
**Pull request by stefanpenner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +89,21 @@ export default Ember.Component.extend({
 });
 ```
 
+#### "readOnly"
+
+```javascript
+import Ember from 'ember';
+import computed from 'ember-computed-decorators';
```
Comment (rwjblue) [inline README.md]: `import computed, { readOnly } from 'ember-computed-decorators';`

File: addon/index.js [hunk 2]
```diff
@@ -41,3 +41,14 @@ export default function computedDecorator(...params) {
     };
   }
 }
+
+export function readOnly(target, name, descriptor) {
+  descriptor.writable = false;
+  var value = typeof descriptor === 'object' && descriptor && descriptor.value;
+
+  if (value instanceof Ember.ComputedProperty) {
```
Comment (rwjblue) [inline addon/index.js]: Can we check `value && value.isDescriptor` instead?

**Comment by stefanpenner** [comment]
oops wait

**Comment by rwjblue** [comment]
Needs a rebase now that #8 is landed.

**Comment by stefanpenner** [comment]
rebased

**Pull request merged**